docs/sphinx/conf.py

```diff
@@ -63,7 +62,7 @@
 # The short X.Y version.
 version = u'0.2'
 # The full version, including alpha/beta/rc tags.
-release = u'0.2.3'
+release = u'0.2.4'
```

Comment:
I think that perhaps we want to use a magic string or something in the place of a version number and then have a release script that updates the number?

Now that we are integrated with readthedocs old versions will be accessible there.